### ISSUE TITLE
load_hnsw in HnswIo panics when loading a hnsw that contains no points

### ISSUE THREAD

**Issue by MathGeniusJodie** [jean-pierreBoth/hnswlib-rs#15]
```rust
let space = Hnsw::<f64, DistL1>::new(
    max_nb_connection,
    max_elements,
    max_layer,
    ef_c,
    DistL1 {},
);
space.file_dump(&file_path);
let mut io = HnswIo::new(PathBuf::from("."), file_path);
io.set_options(ReloadOptions::new(true));
let mmap_space = io.load_hnsw::<f64, DistL1>().unwrap();
```

**Comment by MathGeniusJodie** [comment]
it would be useful to allow this in case you want a persistent data store with no starting data

**Comment by bwsw** [comment]
@MathGeniusJodie fixed in #17
